### DISCUSSION TITLE
2 Jumpers for term, on

### DISCUSSION THREAD

**Discussion by mfaridbadr** [erichelgeson/BlueSCSI-v1#241] [Q&A]
Hi,
In H.W. Ver 1.0c  desktop ver.
what is the need for the 2 jumpers ( term ,on)
and When should it be used? 
and if used them must use both or can use one of them only

Thanks

**Comment by erichelgeson** [comment]
They should be used whenever you need termination on the device - when it is the last device in the chain.

They should both be on or both be off.

**Comment by mfaridbadr** [comment]
@erichelgeson 
then, if the Bluescsi the only one device in the cable, there are no need for them     is that right ?

also , it is not related to get power from berg or scsi cable , if yes  how can i choice the power source

thanks

**Reply by erichelgeson** [reply]
If BlueSCSI is the only device - it is then the last device, so yes you need termination on.

You can power from whatever source gives you a stable 5v - TERMPOWR from the cable, berg, or USB.

**Comment by mfaridbadr** [comment]
@erichelgeson 
Thanks  My Dear

**Comment by mfaridbadr** [comment]
Hi @erichelgeson 
Kindly,   the bluescsi only in the cable 
N.B. : the cable have only 2 sockets   , one connect to Mother board and the other connect to Bluescsi
S.W.  v1.1-20221203
when I connect the 2 jumpers   ,  it shown as 8 device
When i Don't connect the 2 Jumers , it shown as 1 device

![without jumpers](https://github.com/erichelgeson/BlueSCSI/assets/104479896/fd6655f3-8152-4de5-8ccd-95e787fa574f)
![With Jumper](https://github.com/erichelgeson/BlueSCSI/assets/104479896/01318c42-d879-4666-a6f1-310e920b7530)
